AutoGear.lua

```diff
@@ -36,7 +38,9 @@ local weapons
 local tUpdate = 0
 local dataAvailable = nil
 local shouldPrintHelp = false
---local maxPlayerLevel = MAX_PLAYER_LEVEL_TABLE[GetExpansionLevel()]
+if (not IsSL) then
+	local maxPlayerLevel = MAX_PLAYER_LEVEL_TABLE[GetExpansionLevel()]
+end
```

Comment:
Let's replace this 3-line block with:

```local maxPlayerLevel = GetMaxLevelForExpansionLevel(GetExpansionLevel())```

I say this because according to https://wow.gamepedia.com/API_GetMaxLevelForExpansionLevel this function is supported by Classic and BFA too.  I didn't know about it until now.  I'm sure Shadowlands will need to have ```maxPlayerLevel``` populated too.

Reply:
local maxPlayerLevel = GetMaxLevelForExpansionLevel(GetExpansionLevel())
SL - 50
BFA -120
Classic - 60
Shadowlands was /dump on a level 11 Zandalari. "maxPlayerLevel" is only ever used once and it is for experience items which Shadowlands no longer has. 
local maxPlayerLevel = MAX_PLAYER_LEVEL_TABLE[GetExpansionLevel()] Did not work on Shadowlands and is why I restricted it. I will make revisions and recommit. My apologies, GitHub is new to me.

Reply:
No worries.  I appreciate your efforts.  My life has been a whirlwind of getting married, moving, taking care of a newborn baby, and working more than 60 hours per week.  Any help with AutoGear is so awesome.